docker/Dockerfile

```diff
@@ -17,12 +17,16 @@ RUN apt-get update \
     && apt-get install -y apt-transport-https \
     # Get source for node.js
     && curl -fsSL https://deb.nodesource.com/setup_16.x | bash - \
+    # Add PostgreSQL repository
+    && sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list' \
```

Comment:
This is based on https://www.postgresqltutorial.com/postgresql-getting-started/install-postgresql-linux/

Package is not available otherwise.

Reply:
Can you also update Postgres to version 15, which we use in Heroku? Maybe that'll even make thing easier?

Reply:
Side question: should we also update to Node 18, since that's what we use in frontend tests?
https://github.com/mozilla/pontoon/blob/main/.github/workflows/frontend.yml

Reply:
Sounds good.